### PR TITLE
Fix for Capital Letter Entry

### PR DESCRIPTION
**Pull request by bawigga**
See #501 for previous discussion and context.

### CODE REVIEW

File: frontend/termbox/main.go [hunk 1]
```diff
@@ -456,7 +456,7 @@ func (t *tbfe) renderthread() {
 	}
 
 	for range t.dorender {
-		log.Finest("Rendering")
+		// log.Finest("Rendering")
```
Comment (zoli) [inline frontend/termbox/main.go]: Why?

File: backend/sublime/events_manual.go [hunk 2]
```diff
@@ -172,7 +172,7 @@ func (c *OnQueryContextGlue) onQueryContext(v *backend.View, key string, operato
 	defer ret.Decref()
 
 	//	if ret != nil {
-	log.Fine("onQueryContext: %v, %v", pv, ret.Base())
+	// log.Fine("onQueryContext: %v, %v", pv, ret.Base())
```
Comment (zoli) [inline backend/sublime/events_manual.go]: Why commenting out the logs?

**Comment by erbridge** [comment]
LGTM, except for @zoli's comments.

File: frontend/termbox/main.go (hunk 1 above)
Comment (bawigga) [inline frontend/termbox/main.go]: It seemed extraneous and was cluttering up the log output. Happy to add it back in if we feel strongly about it.

File: backend/sublime/events_manual.go (hunk 2 above)
Comment (bawigga) [inline backend/sublime/events_manual.go]: It seemed extraneous and was cluttering up the log output. Happy to add it back in if we feel strongly about it.

**Comment by quarnster** [comment]
LGTM, Commenting the logging out is such a half-fix though :) Either remove the line(s) completely or keep the calls in IMO. I don't feel strongly for either, but I'd like to hear @zoli's opinion first in case he strongly wants them enabled before you patch anything up @bawigga.

File: frontend/termbox/main.go (hunk 1 above)
Comment (bawigga) [inline frontend/termbox/main.go]: This one specifically was being printed out once every second.

**Comment by zoli** [comment]
LGTM, @bawigga I don't feel strongly for either to, Just wanted to know the reason for commenting out and I agree with @quarnster that commenting the logging out is such a half-fix we should remove them if they are cluttering.

**Comment by bawigga** [comment]
@zoli @quarnster thanks for the feedback! Those log outputs have been removed.

**Comment by zoli** [comment]
Thanks.

**Pull request merged**